### ISSUE TITLE
Reduce PDF size

### ISSUE THREAD

**Issue by hmsdoc** [orzih/mkdocs-with-pdf#84]
Compared to the [mkdocs-pdf-export-plugin](https://github.com/zhaoterryy/mkdocs-pdf-export-plugin) I've noticed that the mkdocs-with-pdf plugin generates much larger PDF files. In my test project (which consists of over 170 pages in size A4) this resulted in a file that was about 3 times the size. 

Is there any way to control the PDF output size? Maybe by allowing to reduce the image size (different compression settings) via a configuration parameter?

**Comment by SusanneKrause** [comment]
Same here.
PDF size generated with VS Code Extension: 4.2 MB
PDF size generated with mkdocs-with-pdf: 16.9 MB

Would be great to have the possibility to reduce the file size.

**Comment by pohlt** [comment]
Do you include images multiple times (like logos in the header / footer)? Depending on how they are handled, they might be included multiple times in the resulting PDF which could result in this size difference.

**Comment by hmsdoc** [comment]
@pohlt: No, the images used are all different. Header and footer consist of text only.
